modules/Bio/EnsEMBL/Variation/Utils/VEP.pm

```diff
@@ -631,12 +627,7 @@ sub parse_vcf {
 
         if(defined($type)) {
             # convert to SO term
-            my %terms = (
-                INS  => 'insertion',
-                DEL  => 'deletion',
-                TDUP => 'tandem_duplication',
-                DUP  => 'duplication'
-            );
+            my %terms = %SV_SO_TERMS;
```

Comment:
The keys in `%SV_SO_TERMS` are not the proper VCF types. We would need to run `get_SO_term($type)` to get the correct SO term.